packages/fluentui/e2e/cypress/support/commands.js

```diff
@@ -0,0 +1,97 @@
+const CYPRESS_ACTION_TIMEOUT = 10 * 1000;
+
+Cypress.Commands.add('gotoTestCase', (testFilePath, waitForSelector) => {
+  const { _ } = Cypress;
+  const path = testFilePath
+    .split('/')
+    .pop()
+    .replace(/^(.+).spec.ts?$/, '$1')
+    .replace(/^\//, '');
+
+  cy.visit(`/${_.kebabCase(path)}`);
+  cy.exist(waitForSelector);
+});
+
+Cypress.Commands.add('exist', selector => {
+  cy.get(selector, { timeout: CYPRESS_ACTION_TIMEOUT }).should('exist');
+});
+
+Cypress.Commands.add('notExist', selector => {
+  cy.get(selector, { timeout: CYPRESS_ACTION_TIMEOUT }).should('not.exist');
+});
+
+Cypress.Commands.add('nothingIsFocused', () => {
+  cy.focused().should('not.exist');
+});
+
+Cypress.Commands.add('simulatePageMove', () => {
+  cy.get('body').realTouch();
+});
+
+Cypress.Commands.add('nothingIsFocused', () => {
+  cy.focused().should('not.exist');
+});
+
+Cypress.Commands.add('clickOn', selector => {
+  cy.get(selector).realClick();
+});
+
+Cypress.Commands.add('focusOn', selector => {
+  cy.get(selector).focus();
+});
+
+Cypress.Commands.add('hover', selector => {
+  cy.get(selector).realHover();
+});
+
+Cypress.Commands.add('isFocused', selector => {
+  cy.get(selector, { timeout: CYPRESS_ACTION_TIMEOUT }).should('be.focused');
+});
+
+Cypress.Commands.add('notVisible', selector => {
+  cy.get(selector, { timeout: CYPRESS_ACTION_TIMEOUT }).should('not.be.visible');
+});
+
+Cypress.Commands.add('visible', selector => {
+  cy.get(selector, { timeout: CYPRESS_ACTION_TIMEOUT }).should('be.visible');
+});
+
+Cypress.Commands.add('resizeViewport', width => {
+  cy.viewport(width, Cypress.config('viewportHeight'));
+});
+
+Cypress.Commands.add('expectTextOf', (selector, text) => {
+  cy.get(selector, { timeout: CYPRESS_ACTION_TIMEOUT }).should('have.text', text);
+});
+
+Cypress.Commands.add('expectCount', (selector, count) => {
+  cy.get(selector, { timeout: CYPRESS_ACTION_TIMEOUT }).should('have.length', count);
```

Comment:
nit: looking at the commands following this might be nice to be consistent in naming

```suggestion
Cypress.Commands.add('hasText', (selector, text) => {
  cy.get(selector, { timeout: CYPRESS_ACTION_TIMEOUT }).should('have.text', text);
});

Cypress.Commands.add('hasCount', (selector, count) => {
  cy.get(selector, { timeout: CYPRESS_ACTION_TIMEOUT }).should('have.length', count);
```